ci/datamgr.py

```diff
@@ -477,9 +476,15 @@ def clickhouse(schema, tables, data_directory, **params):
 
 
 @cli.command()
+@click.option(
+    '-S',
+    '--schema',
+    type=click.File('rt'),
+    default=str(SCRIPT_DIR / 'schema' / 'bigquery.sql'),
```

Comment:
can you add a help here (e.g. a schema is a sql file); is ``--schema`` common? e.g. its slightly misleading on what its doing

Reply:
Yes, added.

It's not common to call schema a SQL file in my experience with BigQuery, but it's the same as the other backends are doing. I've added help everywhere this argument is defined.